skin/src/main/java/org/researchstack/skin/AppPrefs.java

```diff
@@ -24,11 +24,16 @@
         prefs = PreferenceManager.getDefaultSharedPreferences(context);
     }
 
-    public static synchronized AppPrefs getInstance(Context context)
+    public static void init(Context context) {
+        instance = new AppPrefs(context);
+    }
+
+    public static synchronized AppPrefs getInstance()
```

Comment:
Yeah, I think we should add call to AppPrefs.getInstance(context) in ResearchStack.init
and I'm always for deprecating over removing in an SDK.